src/AccessibilityInsights.SharedUx/Behaviors/ColumnResizeHotkeyBehavior.cs

```diff
@@ -48,8 +48,19 @@ private void OnColumnResizeHotkeyPress(object sender, RoutedEventArgs e)
             }
             else if (sender is ScannerResultControl srCtrl)
             {
-                srCtrl.HasUserResizedLvHeader = true;
-                ResizeListView(srCtrl.lvDetails);
+                srCtrl.listControl.HasUserResizedLvHeader = true;
+                ResizeListView(srCtrl.listControl);
+            }
+        }
+
+        private static void ResizeListView(ScannerResultCustomListControl lv)
+        {
+            if (lv.View is GridView gv)
+            {
+                foreach (var col in gv.Columns)
+                {
+                    col.Width = Double.NaN;
+                }
```

Comment:
`ScannerResultCustomListControl` exposes `lvDetails`. If we're OK with that pattern, then this can be simplified:
```suggestion
                ResizeListView(srCtrl.listControl.lvDetails);
```

Reply:
I'm ok with it.
Actually it feels more clean, my old implementation was just adding unnecessary extra layers.